### ISSUE TITLE
included savegames are corrupted

### ISSUE THREAD

**Issue by mnalis** [mnalis/ironseed_fpc#34]
for example, `save3` loads  ok, even in old e5216799fcc602cb694647ae297636edac264ef0, but in Log we see 15 messages, but can scroll using keyboard only through first 4. So event count is wrong or something. Probably should be replaced with new default ones created on startup. 

Also, https://github.com/mnalis/ironseed_fpc/issues/23 would need to handle situation with creating new savegames where none were existing before, and similar situations. It could copy them from data dir on startup creation of ~/.ironseed

**Comment by mnalis** [comment]
this also happens with original dos version `ironseed-v1.20.0016-2013-03-17.zip` from `ironseed.net`...

**Comment by nukebloodaxe** [comment]
I'm under the impression, given the content of some of the saves, that they were mainly development build ones.  It's possible they have been generated by hand in some cases to test certain scenarios.

**Comment by mnalis** [comment]
check on what log entry does it stop exactly? is it maybe "0" SENGZHAC? if so, it may be savegames were created in older game version, but new one does not work.

Becase, `comm2.pas:addcursor()` for moving to next log, contains following check:
```
 if (logs[logindex+1]>0) and (logindex<maxlogentries-1) then
```

But, `logs[logindex+1]` will be exactly `0` for `SENGZHAC`, so it would fail. It shoulg check for `-1` which means "no log".
